core/modules/layout/layout.admin.inc

```diff
@@ -281,23 +281,89 @@ function layout_settings_form($form, &$form_state, Layout $layout) {
     }
   }
 
-  $items = array(
+  // Path entry field with dynamic information about the entered path and
+  // toggleable placeholder examples.
+  $path = isset($form_state['values']['path']) ? $form_state['values']['path'] : $layout->getPath();
+  $is_new = isset($layout->is_new) && $layout->is_new;
+  $has_placeholder = strpos($path, '%') !== FALSE;
+  if (empty($path)) {
+    $path_matches_text = t('Please enter a path.');
+  }
+  else {
+    $path_matches = db_query('
+      SELECT path, page_callback
+      FROM {menu_router}
+      WHERE path = :path
+      ', array(':path' => $path))
+      ->fetchAllKeyed();
+    if (count($path_matches)) {
+      if ($has_placeholder) {
+        if ($is_new) {
+          $path_matches_text = t('This layout will override the default layout for existing page(s) whose paths match this pattern.');
+        }
+        else {
+          $path_matches_text = t('This layout overrides the default layout for existing page(s) whose paths match this pattern.');
+        }
+      }
+      else {
+        if ($is_new) {
+          $path_matches_text = t('This layout will override the default layout for an existing page whose path matches this pattern.');
+        }
+        else {
+          $path_matches_text = t('This layout overrides the default layout for an existing page whose path matches this pattern.');
+        }
+      }
+    }
+    else {
+      if ($has_placeholder) {
+        if ($is_new) {
+          $path_matches_text = t('This path will create pages whose paths match this pattern.');
+        }
+        else {
+          $path_matches_text = t('This path provides pages whose paths match this pattern.');
+        }
+      }
+      else {
+        if ($is_new) {
+          $path_matches_text = t('This path will create a page at this path.');
+        }
+        else {
+          $path_matches_text = t('This path provides a page at this path.');
+        }
+      }
+    }
+  }
+  $form['path_matches_text'] = array(
+    '#type' => 'value',
+    '#value' => $path_matches_text,
+  );
+  $path_matches = array(
+    '#type' => 'container',
+    '#access' => !$layout->isDefault(),
+    '#attributes' => array(
+      'id' => array('path-matches'),
+    ),
```

Comment:
I try to avoid IDs for most things these days. Could you make this a class an namespace it under `layout`, like `class="layout-path-matches"`

Reply:
Take a look at function `layout_settings_form_path_ajax()`. All three `ajax_command_html()` calls uses IDs. Why should they not match?

Reply:
Sorry I didn't see that this was part of an AJAX command, in which case I think this is fine :+1:

Might be good to namespace it still, i.e. `layout-path-matches`

Reply:
OK. I did namespace it as you suggested in the latest PR update.